docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,102 @@
+= Run a Sample Cloudflow Application on GCP
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Setting up the Cloudflow `kubectl` Plugin
+
+With Cloudflow installed on a GKE cluster through the Marketplace, you can interact with Cloudflow via its `kubectl` plugin: `kubectl cloudflow`.
+If you haven't already, download and set up the `kubectl` plugin following the https://cloudflow.io/docs/current/get-started/prepare-development-environment.html#_download_and_install_the_cloudflow_cli[instructions].
+
+Then connect your local `kubectl` to the GKE cluster on which Cloudflow is installed. You can find the command to do that from the "Clusters" view:
+
+.Connect `kubectl` to GKE cluster
+image::connect-clusters.png[Connect kubectl to GKE cluster]
+
+It should look like the following:
+[source,bash]
+----
+$ gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <gcp-project>
+----
+
+== Building and Deploying call-record-aggregator
+
+Next we'll deploy a sample Cloudflow application called https://github.com/lightbend/cloudflow/tree/master/examples/call-record-aggregator[call-record-aggregator] to the cluster.
+Clone the Cloudflow https://github.com/lightbend/cloudflow[Github repo] and `cd` into the `examples/call-record-aggregator` directory. Then run the following command to build the application:
+
+[source,bash]
+----
+$ sbt buildApp
```

Comment:
This step fails for me with: 
```
[info] denied: requested access to the resource is denied
[error] java.lang.RuntimeException: Failed to push
[error] 	at scala.sys.package$.error(package.scala:30)
[error] 	at sbtdocker.DockerPush$.apply(DockerPush.scala:42)
[error] 	at sbtdocker.DockerPush$.$anonfun$apply$1(DockerPush.scala:17)
[error] 	at sbtdocker.DockerPush$.$anonfun$apply$1$adapted(DockerPush.scala:16)
[error] 	at scala.collection.immutable.List.foreach(List.scala:392)
[error] 	at sbtdocker.DockerPush$.apply(DockerPush.scala:16)
[error] 	at sbtdocker.DockerSettings$.$anonfun$baseDockerSettings$2(DockerSettings.scala:24)
[error] 	at sbtdocker.DockerSettings$.$anonfun$baseDockerSettings$2$adapted(DockerSettings.scala:19)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
[error] 	at sbt.Execute.work(Execute.scala:290)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
```
I'm wondering if I haven't setup the Docker or SBT correctly to point to / authenticate to Google's image registry. Has that been mentioned or covered before this?

Reply:
The user needs to authenticate docker with the registry specified in `call-record-aggregator/target-env.sbt`. If you don't have that file, you can create one based on the example provided in `target-env.sbt.example`. This is probably documented somewhere else. But for the sake of completeness, I'll mention it here as well.

Reply:
If it's documented elsewhere, can link to it above, otherwise, I think it's best to include it as a step here.

Reply:
I think I got this working now - we definitely want to include how to setup target-env.sbt in this doc though.

This step is taking a long time to complete. I think it's worth adding a "This step may take several minutes to complete, as it will build and push multiple Docker images to your Google Kubernetes cluster." or similar.

Reply:
Sounds good. Will add.